google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/AbstractITDatastoreTest.java

```diff
@@ -1563,6 +1568,285 @@ public void testQueryWithReadTime() throws InterruptedException {
     }
   }
 
+  @Test
+  public void testSumAggregation() {
+    datastore.put(AGGREGATION_ENTITY_1, AGGREGATION_ENTITY_2);
+
+    EntityQuery baseQuery = Query.newEntityQueryBuilder().setKind(MARKS_KIND).build();
+    AggregationQuery aggregationQuery =
+        Query.newAggregationQueryBuilder()
+            .over(baseQuery)
+            .addAggregations(sum("marks").as("total_marks"))
+            .setNamespace(NAMESPACE)
+            .build();
+
+    // sum of 2 entities
+    assertThat(getOnlyElement(datastore.runAggregation(aggregationQuery)).getLong("total_marks"))
+        .isEqualTo(184L);
```

Comment:
Nit: Since these long int values "184L" and "239L" are used in multiple test cases, can we declare these as constants in the class for reusability?

Reply:
I see your point, but because this is only used in an assertion and not part of the test logic itself, it may be more difficult from a readability perspective to have to check back at the variable name at the top of the file.